### PR TITLE
add mapDamage to sarek

### PR DESCRIPTION
**Pull request by chelauk**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies:
   - bioconda::genesplicer=1.0
   - bioconda::htslib=1.9
   - bioconda::manta=1.6.0
+  - bioconda::mapdamage2=2.2.0=pyr36_1
```
Comment (MaxUlysse) [inline environment.yml]: What is this part at the end of the version number? `=pyr36_1`

Comment (chelauk) [reply environment.yml]: I copied it from the mapDamage [installation instructions](https://ginolhac.github.io/mapDamage/)

Comment (MaxUlysse) [reply environment.yml]: I see, It's making the linting fail unfortunately.
And I'm noticing now that it's needing `python` > `3.5`, I'm afraid we'll get compatibilities issues with that.

Comment (chelauk) [reply environment.yml]: I've just had a look at anaconda and conda install -c bioconda mapdamage2 will should work

Comment (MaxUlysse) [reply environment.yml]: OK, can you try and see if you can build the docker container with `mapdamage2` on your machine?

**Pull request closed without merging**